src/components/popover/popover.tsx

```diff
@@ -148,8 +148,8 @@ export class Popover {
     }
 
```

Comment:
shouldn't we remove things like 
```tsx
        const popoverZIndex = getComputedStyle(this.host).getPropertyValue(
            '--popover-z-index',
        );
```
already in this PR?

Reply:
We probably can

Reply:
⚡